pallets/automation-time/src/tests.rs

```diff
@@ -231,19 +236,92 @@ fn schedule_native_transfer_works() {
 	})
 }
 
+#[test]
+fn schedule_xcmp_works() {
+	new_test_ext(START_BLOCK_TIME).execute_with(|| {
+		let call: Vec<u8> = vec![2, 4, 5];
+		get_funds(Sibling::from(PARA_ID).into_account());
+		assert_ok!(AutomationTime::schedule_xcmp_task(
+			cumulus_pallet_xcm::Origin::SiblingParachain(PARA_ID.try_into().unwrap()).into(),
+			vec![50],
+			vec![SCHEDULED_TIME],
+			PARA_ID.try_into().unwrap(),
+			call.clone(),
+			100_000,
+		));
+		match AutomationTime::get_scheduled_tasks(SCHEDULED_TIME) {
```

Comment:
wonder if it's worthwhile checking the tasks map as well, to make sure it's correctly typed as type "XCMP" rather than normal notify or something else

Reply:
This test does that.  It gets all the scheduled taskId's, which should only be 1, then it grabs the task from the task map and compares it to what we expect it to be.

Reply:
gah, if i just read that more closely. i misread the next 2 lines 🤦